docs/src/Table.doc.js

```diff
@@ -24,6 +24,11 @@ card(
         name: 'children',
         type: 'React.Node',
       },
+      {
+        name: 'hasBorder',
```

Comment:
@PayalModi instead of a boolean prop, could we pass in `borderSize` and limit it to `sm` for now? Gives us more flexibility moving forward + maintains a consistent API with the rest of Gestalt (e.g. `Box`)

Reply:
Yes, I'll update it to be borderSize that allows 'sm' | 'none' to keep it the same as Box